### PR TITLE
docs: update price-feeder readme config section

### PR DESCRIPTION
**Pull request by natexlee**
## Description

closes: #1039

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by natexlee — COMMENTED** [review]
Initial definitions/overview of the three new options.

File: price-feeder/README.md [hunk 1]
```diff
@@ -48,6 +48,20 @@ $ price-feeder /path/to/price_feeder_config.toml
 ```
 
 ## Configuration
+There are multiple new options for validators to setup in the price feeder config, the options are telemetry, deviation, and custom endpoints. 
+
+### `Telemetry`
+• A set of options for the /metrics endpoint when running the price feeder.
```
Comment (adamewozniak) [inline price-feeder/README.md:54]: Let's try to be a bit more descriptive, use the same case they would in the config file (e.g., `deviation_thresholds` or `telemetry`, and add the same spacing we're using on other parts of the config list:
```suggestion
### `telemetry`

A set of options for the application's telemetry, which is disabled by default. An in-memory sink is the default, but Prometheus is also supported.
```

Maybe it'd be helpful to also link to the config here?

File: price-feeder/README.md [hunk 2]
```diff
@@ -48,6 +48,20 @@ $ price-feeder /path/to/price_feeder_config.toml
 ```
 
 ## Configuration
+There are multiple new options for validators to setup in the price feeder config, the options are telemetry, deviation, and custom endpoints. 
+
+### `Telemetry`
+• A set of options for the /metrics endpoint when running the price feeder. 
+
+### `Deviation`
+• Deviation allows validators to set a custom amount of standard deviations around the median which helps distinguish providers from other providers.
```
Comment (adamewozniak) [inline price-feeder/README.md:57]: Add a note about how the default is for this is 1 standard deviation. 

Also, it's not for distinguishing providers from one another; this "deviation allowance" is supposed to help if any providers became faulty.

**Review by adamewozniak — CHANGES_REQUESTED** [review]
Good start! Please update your PR title to adhere to the commit type syntax https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json

**Review by natexlee — COMMENTED** [review]
Documentation

**Review by adamewozniak — COMMENTED** [review]
Almost there, also need to update your PR title. Something like: 
docs: update price-feeder readme config section

**Review by natexlee — COMMENTED** [review]
Changes requested by Woz

File: price-feeder/README.md [hunk 3]
```diff
@@ -49,6 +49,21 @@ $ price-feeder /path/to/price_feeder_config.toml
 
 ## Configuration
 
+### `telemetry`
+
+A set of options for the application's telemetry, which is disabled by default. An in-memory sink is the default, but Prometheus is also supported.
+```shell
+$ price-feeder /path/to/price_feeder_config.toml
+```
```
Comment (robert-zaremba) [inline price-feeder/README.md:57]: please add a link to cosmos-sdk telemetry: https://github.com/cosmos/cosmos-sdk/blob/main/docs/core/telemetry.md

**Review by robert-zaremba — COMMENTED** [review]
To be honest, I think it's better to have that documentation in the config file comments - it will be easier to maintain it.

**Review by natexlee — COMMENTED** [review]
added the link requested by Robert

**Review by natexlee — COMMENTED** [review]
fixed trailing spaces

**Review by adamewozniak — APPROVED** [review]
LGTM! Great job

**Pull request merged**